consul_exporter.go

```diff
@@ -140,6 +146,20 @@ func (e *Exporter) Collect(ch chan<- prometheus.Metric) {
 		clusterServers, prometheus.GaugeValue, float64(len(peers)),
 	)
 
+	leader, err := e.client.Status().Leader()
+	if err != nil {
+		log.Errorf("Query error is %v", err)
+	}
+	if len(leader) == 0 {
+		ch <- prometheus.MustNewConstMetric(
+			clusterLeader, prometheus.GaugeValue, 0,
+		)
+	} else {
+		ch <- prometheus.MustNewConstMetric(
+			clusterLeader, prometheus.GaugeValue, 1,
```

Comment:
Not really; I considered that, but though people might be confused if it was a number other than 0 or 1 - I can image questions like "exporter says I have 12 leaders in a 3 node cluster".